blogs/secondary_interface/secondary_interface.md

```diff
@@ -0,0 +1,155 @@
+   for network interfaces in non-default network namespaces. This step is crucial for
+   monitoring and controlling network traffic within these namespaces.
+
+Here is a sample configuration to build and test this topology
+
+- Install the SRIOV operator in the environment.
+- Identify the SRIOV capable device on the node.
+- Label the node which has SRIOV interface with the label
+  `feature.node.kubernetes.io/network-sriov.capable=true`
+- Create the SriovNetworkNodePolicy object
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: mypolicy
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: netdeviceresource
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  priority: 99
+  numVfs: 50
+  nicSelector:
+    pfNames: ["ens7f0np0#25-49"]
+  deviceType: netdevice
+```
+
+- Create the SriovNetwork object. This will create net-attach-def in
+  `openshift-sriov-network-operator` namespace
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-test
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: netdeviceresource
+  networkNamespace: test
+  ipam: '{ "type": "static", "addresses": [{"address": "192.168.122.71/24"}]}'
+```
+
+- Create a test pod using `SRIOVNetwork` created above denoted by annotation
+  `k8s.v1.cni.cncf.io/networks: "sriov-test"`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: httpd-2
+  namespace: openshift-sriov-network-operator
+  labels:
+    app: sriov
+  annotations:
+    k8s.v1.cni.cncf.io/networks: "sriov-test"
+spec:
+  containers:
+  - name: httpd
+    command: ["sleep", "30d"]
+    image: registry.redhat.io/rhel8/support-tools
+    ports:
+    - containerPort: 8080
+    securityContext:
+      allowPrivilegeEscalation: false
+      seccompProfile:
+        type: RuntimeDefault
+      capabilities:
+        drop:
+        - ALL
+```
+
+- Deploy network observability operator
+
+- Create `FollowCollector` object with `privileged` set
+
+```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    type: EBPF
+    ebpf:
+      privileged: true
+```
+
+- Network observability raw flows filtered on Pod VF interface `net1`
+
+```json
+{
+  "AgentIP": "10.73.116.62",
+  "Bytes": 406,
+  "DstAddr": "192.168.122.72",
+  "DstMac": "06:D8:93:DB:56:9A",
+  "DstPort": 60256,
+  "Duplicate": false,
+  "Etype": 2048,
+  "Flags": 784,
+  "FlowDirection": "1",
+  "IfDirection": 1,
+  "Interface": "net1",
+  "K8S_ClusterName": "5846f979-9d5b-4a1b-b08d-aee52e6b58a2",
+  "Packets": 4,
+  "Proto": 6,
+  "SrcAddr": "192.168.122.71",
+  "SrcMac": "AE:39:F4:7E:03:FE",
+  "SrcPort": 8080,
+  "TimeFlowEndMs": 1694120614010,
+  "TimeFlowStartMs": 1694120614009,
+  "TimeReceived": 1694120618,
+  "app": "netobserv-flowcollector"
+}
+```
```

Comment:
At the end, here, I'd reiterate that you can do all the above, or you can just install network observability operator 1.4 or higher and have it all done for you -- and maybe point to our repositories or something else -- I think there's a great opportunity here to promote the work you've done in the context of network obs. and maybe get more people to try

Reply:
hmm I think there is misunderstanding , we still need to provision SRIOV, SRIOV policy etc that hasn't changed what is new here is with turning on the operator know and the new work done in the agent in 1.4 we know can see those flows

Reply:
I will add the repo and feedback sections like in another blogs